app/templates/maybe-eligible.njk

```diff
@@ -48,6 +48,10 @@
       {% endif %}
       <p class="govuk-body">{{ messageContent | safe}}</p>
 
+        {% if isconcreteBunkerSize %}
+            <p class="govuk-body">The grant contribution for the concrete bunker is capped at £16,180.</p>
```

Comment:
again here, could we use a yar value for £16,100? just in case the value changes so we dont need to rely on hardcoding